pom.xml

```diff
@@ -8,7 +8,7 @@
 
   <groupId>com.cloudbees.jenkins.plugins</groupId>
   <artifactId>docker-custom-build-environment</artifactId>
-  <name>CloudBees Docker Custom Build Environment Plugin</name>
+  <name>Docker Custom Build Environment Plugin. Maintained by CloudBees, Inc.</name>
```

Comment:
@jglick I did what we did in other plugins. If the criterial have changed... It is fine for me. I'll update the PR.